contracts/seed/Seed.sol

```diff
@@ -117,7 +114,6 @@ contract Seed {
     }
 
     modifier allowedToWithdraw() {
-        require(!paused, "Seed: should not be paused");
```

Comment:
Hmm, why is this line being removed?

Reply:
Because this is adding an extra step to withdraw. admin can anytime pause and unpause and withdraw.

Reply:
Hmm, but this is locking a withdrawal for everyone if contract is paused... 
I'm not sure about the requirements, but if you say so...